include/class.forms.php

```diff
@@ -3083,6 +3078,10 @@ function setChoices($choices) {
         $this->_choices = $choices;
     }
 
+    function display($value) {
+        return $this->getAnswer()->value;
```

Comment:
Wouldn't this crash if a field is not a DynamicField?  That said, this should call display on answer and also note that an answer might not exist yet on a particular object e.g new field gets added to Ticket Details form. System auto-creates form field answer when getting dynamic fields by adding missing fields but it's not a safe assumption to depend on.

Also curious where this is getting called from - what might `$value` be?

```php

function display($value) {
    if (($a=$this->getAnswer())
        return $a->display();

    return Format::htmlchars((string) $value);
} 
```